<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite ignored="12" hostname="LAPTOP-HKP3MAF8" failures="0" tests="1" name="qaf.example.tests.CreateAccountFailTest" time="3.244" errors="1" timestamp="07 Jun 2021 10:07:44 GMT" skipped="0">
  <testcase classname="qaf.example.tests.CreateAccountFailTest" name="createAccountFailTest" time="3.244">
    <error message="class java.lang.Long cannot be cast to class java.lang.String (java.lang.Long and java.lang.String are in module java.base of loader &#039;bootstrap&#039;)" type="java.lang.ClassCastException">
      <![CDATA[java.lang.ClassCastException: class java.lang.Long cannot be cast to class java.lang.String (java.lang.Long and java.lang.String are in module java.base of loader 'bootstrap')
	at qaf.example.tests.CreateAccountFailTest.createAccountFailTest(CreateAccountFailTest.java:28)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- createAccountFailTest -->
  <testcase classname="com.qmetry.qaf.automation.testng.TestNGTestCase" name="setupTest">
    <skipped/>
  </testcase> <!-- setupTest -->
  <testcase classname="com.qmetry.qaf.automation.testng.TestNGTestCase" name="setupTest">
    <skipped/>
  </testcase> <!-- setupTest -->
  <testcase classname="com.qmetry.qaf.automation.testng.TestNGTestCase" name="setupTest">
    <skipped/>
  </testcase> <!-- setupTest -->
  <testcase classname="com.qmetry.qaf.automation.testng.TestNGTestCase" name="setupTest">
    <skipped/>
  </testcase> <!-- setupTest -->
  <testcase classname="com.qmetry.qaf.automation.testng.TestNGTestCase" name="setupSuit">
    <skipped/>
  </testcase> <!-- setupSuit -->
  <testcase classname="com.qmetry.qaf.automation.testng.TestNGTestCase" name="setupSuit">
    <skipped/>
  </testcase> <!-- setupSuit -->
  <testcase classname="com.qmetry.qaf.automation.testng.TestNGTestCase" name="setupSuit">
    <skipped/>
  </testcase> <!-- setupSuit -->
  <testcase classname="com.qmetry.qaf.automation.testng.TestNGTestCase" name="setupSuit">
    <skipped/>
  </testcase> <!-- setupSuit -->
  <testcase classname="com.qmetry.qaf.automation.ui.AbstractTestCase" name="afterTest">
    <skipped/>
  </testcase> <!-- afterTest -->
  <testcase classname="com.qmetry.qaf.automation.ui.AbstractTestCase" name="afterTest">
    <skipped/>
  </testcase> <!-- afterTest -->
  <testcase classname="com.qmetry.qaf.automation.ui.AbstractTestCase" name="afterTest">
    <skipped/>
  </testcase> <!-- afterTest -->
  <testcase classname="com.qmetry.qaf.automation.ui.AbstractTestCase" name="afterTest">
    <skipped/>
  </testcase> <!-- afterTest -->
</testsuite> <!-- qaf.example.tests.CreateAccountFailTest -->
